foyer-intrusive/src/lib.rs

```diff
@@ -16,6 +16,7 @@
 #![feature(ptr_metadata)]
 #![feature(trait_alias)]
 #![feature(lint_reasons)]
+#![feature(offset_of)]
```

Comment:
Is the feature used anywhere? If not, please remove it.

Reply:
The compiler told me to add this because the `instrusive_xxx!` use the unstable feature `offset_of`.

Reply:
Oh, i get it.